[quant] Move quantization tests into test/quantization

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35721 [quant][graphmode][refactor] Move the whitelists to a centeralized place
* **#35688 [quant] Move quantization tests into test/quantization**

Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

Differential Revision: [D20771828](https://our.internmc.facebook.com/intern/diff/D20771828)

jerryzh168 committed:
[quant] Move quantization tests into test/quantization

Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar reviewed (approved):
Looks good. Do we need to update internal TARGETS file as well?

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
I think so, will do

jerryzh168 committed:
Update on "[quant] Move quantization tests into test/quantization"


Summary:

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

Differential Revision: [D20771828](https://our.internmc.facebook.com/intern/diff/D20771828)

[ghstack-poisoned]

suo commented:
@jerryzh168 I am unlanding this PR as it broke master. Looks like the CI failed here and the Dr CI commenter also detected this issue. Please check these before you land in the future :)
